apps/crn-server/src/data-providers/contentful/reminder.data-provider.ts

```diff
@@ -719,7 +719,7 @@ const getInReviewResearchOutputRemindersFromQuery = (
       !researchOutput.documentType ||
       !isResearchOutputDocumentType(researchOutput.documentType) ||
       isPublished ||
-      !researchOutput.reviewRequestedBy
+      !researchOutput.statusChangedBy
```

Comment:
I believe we should use `!researchOutput.isInReview` here as well. This is the `getInReviewResearchOutputRemindersFromQuery` and although `statusChangedBy` can be populated, this will not guarantee that the status is "in review", right?

Reply:
No @andreicovaciu, because the filter we're using here is this 

```
export const getResearchOutputFilter = (
  zone: string,
): ResearchOutputsFilter => {
  const { last24HoursISO } = getReferenceDates(zone);
  return {
    OR: [
      {
        AND: [
          { addedDate_gte: last24HoursISO },
          { sys: { publishedVersion_exists: true } },
        ],
      },
      {
        AND: [
          { createdDate_gte: last24HoursISO },
          { sys: { publishedVersion_exists: false } },
          { addedDate_exists: false },
          { reviewRequestedBy_exists: false },
        ],
      },
      {
        AND: [
          { sys: { publishedVersion_exists: false } },
          { addedDate_exists: false },
          { reviewRequestedBy_exists: true },
        ],
      },
    ],
  };
};
```

It fetches all 3 types of research outputs reminders and in the functions `getPublishedResearchOutputRemindersFromQuery`, `getDraftResearchOutputRemindersFromQuery` and `getInReviewResearchOutputRemindersFromQuery` we separate them